chunky/src/java/se/llbit/chunky/chunk/biome/SimpleQuartBiomeData3d.java

```diff
@@ -0,0 +1,44 @@
+package se.llbit.chunky.chunk.biome;
+
+import java.util.Arrays;
+
+/**
+ * Implementation of a 4*64*4 biome representation
```

Comment:
Maybe explain what's the difference to `GenericQuartBiomeData3d` and for which chunk versions it is relevant (see comment above)?